README.md

```diff
@@ -1,4 +1,9 @@
-## About
```

Comment:
Please do not change the staged-recipes README. The `summa` feedstock readme will be autogenerated from the recipe. All you need to do is to submit the recipe.

Reply:
reverted